### ISSUE TITLE
Send the booking details to the given e-mail address

### ISSUE THREAD

**Issue by jhsi517** [SE701ProjectGroup4/git-brunching#275]
**User Story:**

As a user, I want the booking ID to be sent to my email, so I can find my booking details without writing them down.

**Acceptance Criteria:**
- [x] User receive an email that contains booking details when booked successfully

**Additional Context:**
Might be a good idea to raise another ticket that checks the validity of the given email address once this is done/closed

**Comment by mitchfaulcon** [comment]
Sounds good to me
